docs/calicoctl/config.md

```diff
@@ -0,0 +1,160 @@
+
+# User guide for 'calicoctl config' commands
+
+This sections describes the `calicoctl config` commands.
+
+The `calicoctl config` command allows users to view or modify 
+low-level component configurations for Felix and BGP.
+
+Read the [calicoctl command line interface user guide](../calicoctl.md) for a full list of calicoctl commands.
+
+## Displaying the help text for 'calicoctl config' commands
+
+Run `calicoctl config --help` to display the following help menu for the 
+calicoctl config commands.
+
+```
+
+Usage:
+  calicoctl config felix <NAME> [<VALUE>|--remove] [--force]
+  calicoctl config bgp <NAME> [<VALUE>|--remove] [--force]
+  calicoctl config node bgp <NAME> [<VALUE>|--remove] [--force]
+
+Description:
+  Configure or show low-level component configuration for Felix and BGP.
+
+Options:
+ --remove  Remove the configuration entry.
+ --force   Force update of configuration entry even if key value is unknown,
+           or the value is not recognized as valid.
+
+Valid configuration:
+  Command         | <NAME>   | <VALUE>s
+------------------+----------+-----------------------------------------
+  config felix    | loglevel | none debug info warning error critical
+  config bgp      | loglevel | none debug info
+  config node bgp | loglevel | none debug info
+
+Warnings:
+  -  Changing the global BGP logging levels using the `calicoctl config bgp`
+     command may cause all BGP sessions to bounce potentially resulting in a
+     transient loss of service.  If you need to change the logging level for
+     BGP, it is recommended to change the levels on a per-node basis using
+     the `calicoctl config node bgp` command.
+
+```
+
+## calicoctl config commands
+
+
+### calicoctl config felix <NAME> 
+This command allows you to show or modify key values for configuration 
+associated with the Felix process.
+
+Currently, you can modify the following:
+```
+  Command         | <NAME>   | <VALUE>s
+------------------+----------+-----------------------------------------
+  config felix    | loglevel | none debug info warning error critical
+```
+
+This command can be run on any Calico node.
+
+Command syntax:
+
+```
+calicoctl config felix <NAME> [<VALUE>|--remove] [--force]
+
+    <NAME>: Config variable key in question.
+    <VALUE>: Value to assign to the config variable.
+
+    --remove: Remove the config key value.
```

Comment:
Need to explain why you would want to remove the configuration.  I believe the global config here overrides anything in the Felix config file.  You should confirm that's true, and then write some text.